presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoNativeIcebergTpcdsQueriesOrcUsingThrift.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.testing.ExpectedQueryRunner;
+import com.facebook.presto.testing.QueryRunner;
+import org.testng.annotations.Test;
+
+@Test(groups = {"orc"})
+public class TestPrestoNativeIcebergTpcdsQueriesOrcUsingThrift
+        extends AbstractTestNativeTpcdsQueries
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        this.storageFormat = "ORC";
+        return PrestoNativeQueryRunnerUtils.createNativeIcebergQueryRunner(true, "ORC");
+    }
+
+    @Override
+    protected ExpectedQueryRunner createExpectedQueryRunner()
+            throws Exception
+    {
+        this.storageFormat = "ORC";
+        return PrestoNativeQueryRunnerUtils.createJavaIcebergQueryRunner("ORC");
+    }
+
+    protected void runAllQueries() throws Exception
+    {
+        testTpcdsQ1();
+        testTpcdsQ2();
+        testTpcdsQ3();
+        testTpcdsQ4();
+        testTpcdsQ5();
+        testTpcdsQ6();
+        testTpcdsQ7();
+        testTpcdsQ8();
+        testTpcdsQ9();
+        testTpcdsQ10();
+        testTpcdsQ11();
+        testTpcdsQ12();
+        testTpcdsQ13();
+        testTpcdsQ14_1();
+        testTpcdsQ14_2();
+        testTpcdsQ15();
+        testTpcdsQ16();
+        testTpcdsQ17();
+        testTpcdsQ18();
+        testTpcdsQ19();
+        testTpcdsQ20();
+        // testTpcdsQ21();
```

Comment:
Do we need to comment here given we have a check inside AbstractTestNativeTpcdsQueries?

Reply:
+1. Nice catch. Yeah these can be avoided.

Reply:
@wypb : Seems like there was a misunderstanding about this review comment. 

Since testTpcdsQ21 has the following condition
```
        if (!storageFormat.equals("ORC")) {
            assertQuery(session, getTpcdsQuery("33"));
        }
```

then it should pass in runAllQueries. We don't need to comment it in this function. We can uncomment the test call at this point.

Reply:
Got it, I will refactor the code

Reply:
@majetideepak @aditi-pandit I have moved `runAllQueries()` from `TestPrestoNativeIcebergTpcdsQueriesOrcUsingThrift.java` and `TestPrestoNativeIcebergTpcdsQueriesParquetUsingThrift.java` to `AbstractTestNativeTpcdsQueries.java`.